app-i18n/man-pages-fr/files/man-pages-fr-3.70-r1-fix-unpack.patch

```diff
@@ -0,0 +1,252 @@
+diff --git a/Makefile.old b/Makefile
+index a4e4285..11844b6 100644
+--- a/Makefile.old
++++ b/Makefile
+@@ -8,88 +8,30 @@ WORK_DIR = $(shell pwd)
+ export LANGS
+ export WORK_DIR
+ 
+-unpack: stamp-unpack
```

Comment:
Don't remove more than necessary, this only makes the patch unnecessarily big. Just modify the dependencies or make calls not to call the targets.

That said, with such intrusive changes to Makefile, I suppose some explanation why you need to do them would be helpful.